packages/pipeline/src/entities/slippage.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+import { BigNumber } from '@0x/utils';
+
+import { bigNumberTransformer, numberToBigIntTransformer } from '../utils';
+
+@Entity({ name: 'slippage', schema: 'raw' })
+export class Slippage {
+    @PrimaryColumn({ name: 'observed_timestamp', type: 'bigint', transformer: numberToBigIntTransformer})
```

Comment:
The set of primary columns in this file is different from the set declared in the migration script.  Change one or the other, so that they're consistent.